### ISSUE TITLE
Importing a Malformed `animj` File Does Not Show Any In-game Error Message

### ISSUE THREAD

**Issue by stiefeljackal** [Yellow-Dog-Man/Resonite-Issues#2452]
### Describe the bug?

When importing an `animj` file that contains malformed JSON such as extra special characters or missing characters (i.e., comma, colon, left or right brace), nothing happens in the current session to indicate an error has occurred. The user needs to look in the logs to locate the error.

### To Reproduce

1. Create an `animj` file with malformed JSON or download the provided file located in the zip ([test_malformed.animj.zip](https://github.com/user-attachments/files/16061601/test_malformed.animj.zip)).
2. Import the `animj` file into Resonite.

### Expected behavior

The expectation is that a text content panel should spawn with the text of the error. This would be something very similar to importing an avatar model with missing bones or importing a LUT with an incompatible size.

### Resonite Version Number

2024.6.25.1149

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Index

### Log Files

[JACKAL_MK2 - 2024.6.25.1149 - 2024-07-02 00_03_45.log](https://github.com/user-attachments/files/16061612/JACKAL_MK2.-.2024.6.25.1149.-.2024-07-02.00_03_45.log)

### Reporters

StiefelJackal / @stiefel

**Comment by stiefeljackal** [comment]
@shiftyscales Can you explain why this was recategorized as an enhancement instead of a bug?

Currently, there are error dialogs that do appear when someone imports a bad asset such as importing an avatar model with missing bones or importing a LUT that is too small. Additionally, an exception is thrown and recorded in the log at `12:05:22 AM.624`. This is an actual error response that should be presented to the user without the user having to dig through the log file to figure out what went wrong. To put this in another perspective, if I submitted an issue where a submitted login form does not show any validation errors or does not respond with why my login failed, would that be considered an enhancement?

**Comment by Frooxius** [comment]
@stiefeljackal Generally we classify "bugs" as features that are implemented, but are malfunctioning in some way. 

In this case, this would be a bug, if the error message indicator was implemented to happen for animj files, but wasn't showing for some reason.

In this case, it's a feature that doesn't exist in the first place and needs to be added. We wouldn't consider lack of a feature a bug in most cases. Therefore we consider it an enhancement.

In short:
Bug - existing feature/behavior that is broken
Enhancement - adding a new feature that doesn't exist

**Comment by Frooxius** [comment]
@stiefeljackal You necessarily wouldn't - we don't expect users to be able to make distinction in all cases - that's why we look at the issue and re-categorize it appropriately if necessary.

**Comment by mpmxyz** [comment]
I'd like to add to this issue that AnimJ loading is also very lenient with missing or excess properties. (up to the point where it is masking missing support for matrix types #2478)
